xbmc/settings/DisplaySettings.cpp

```diff
@@ -655,12 +655,17 @@ RESOLUTION CDisplaySettings::GetResolutionForScreen()
   return RES_DESKTOP;
 }
 
+static inline bool ModeSort(std::pair<std::string, std::string> i,std::pair<std::string, std::string> j)
+{
+  return (i.second > j.second);
```

Comment:
Did you intend to sort from least to greatest or greatest to least?

Reply:
This sort greatest to least